kroxylicious-filters/pom.xml

```diff
@@ -50,6 +50,7 @@
                                     </excludes>
                                     <includes combine.children="append">
                                         <include>io.kroxylicious:kroxylicious-annotations</include>
+                                        <include>io.kroxylicious:kroxylicious-authorizer</include>
```

Comment:
Did a rename get missed? I think this is supposed to be `kroxylicious-authorizer-api` or maybe `kroxylicious-authorizer-acl`